lib/hooks/gitlab/templates/pipeline.html.haml

```diff
@@ -0,0 +1,37 @@
+:ruby
+  label_classes = {
+    'failed'  => 'label-danger',
+    'created' => 'label-info',
+    'manual'  => 'label-info',
+    'running' => 'label-info',
+    'success' => 'label-success',
+    'success_with_warnings' => 'label-warning'
+  }
+  web_url = payload.project.web_url
+  p_status = payload.object_attributes.status
+  p_label_class = label_classes[p_status].presence || 'label-default'
+  p_id = payload.object_attributes.id
+  p_url = "#{web_url}/pipelines/#{p_id}"
+
+%div
+  %h4
```

Comment:
This h4 makes the message relatively larger than other Gitlab messages and maybe a little awkward UI. Could we remove this? (I'm sorry the screenshots I posted were too large and misleading :pray: )

![selection_301](https://user-images.githubusercontent.com/43346/53468427-7fce1700-3a9d-11e9-9a48-8099924c1aff.png)

Reply:
Yes. I removed it.